src/LinuxTracing/PerfEvent.h

```diff
@@ -262,12 +272,13 @@ struct PerfEvent {
   PerfEventOrderedStream ordered_stream = PerfEventOrderedStream::kNone;
   std::variant<ForkPerfEventData, ExitPerfEventData, LostPerfEventData, DiscardedPerfEventData,
                StackSamplePerfEventData, CallchainSamplePerfEventData, UprobesPerfEventData,
-               UprobesWithArgumentsPerfEventData, UretprobesPerfEventData,
-               UretprobesWithReturnValuePerfEventData, UserSpaceFunctionEntryPerfEventData,
-               UserSpaceFunctionExitPerfEventData, MmapPerfEventData,
-               GenericTracepointPerfEventData, TaskNewtaskPerfEventData, TaskRenamePerfEventData,
-               SchedSwitchPerfEventData, SchedWakeupPerfEventData, AmdgpuCsIoctlPerfEventData,
-               AmdgpuSchedRunJobPerfEventData, DmaFenceSignaledPerfEventData>
+               UprobesWithArgumentsPerfEventData, UprobesWithStackPerfEventData,
```

Comment:
I think the order here reflects the order of definition of the `struct`s, so let's be consistent with that. I would suggest defining `UprobesWithStackPerfEventData` after `UprobesWithArgumentsPerfEventData`.